wp-content/themes/core/assets/css/src/theme/layouts/default.pcss

```diff
@@ -71,7 +71,7 @@ main {
 
 .l-sink {
 
-	& > *:not(.c-block):not(.alignwide):not(.alignfull) {
+	& > *:not(.c-block):not(.alignwide):not(.alignfull):not(.b-spacer) {
```

Comment:
I worry about adding additional conditions here as it's already more gnarly than it should be. Would it be possible to just use one the the existing classes (`alignwide` or `alignfull`, maybe) to accommodate this? (Here and on line 101)

Reply:
Good call. I will give that a go